### ISSUE TITLE
Cannot open shared object file in 18.04 (libFoundation.so)

### ISSUE THREAD

**Issue by sglushchenko** [IBM-Swift/swift-ubuntu-docker#60]
Hello,
I have similar problem with #56 . Maybe I made something wrong.
I try to launch a binary release swift file.

My Dockerfile:
`
FROM ibmcom/swift-ubuntu-runtime:latest
COPY . /app
CMD ["/app/server"]
`

I have error:
`
/app/server: error while loading shared libraries: libFoundation.so: cannot open shared object file: No such file or directory
`

Can you help with it?

**Comment by sglushchenko** [comment]
I found some problem.
You have Docker containers `ibmcom/swift-ubuntu` and `ibmcom/swift-ubuntu-runtime`. But `ibmcom/swift-ubuntu` has `Ubuntu Ubuntu 16.04.5 LTS`, and `ibmcom/swift-ubuntu-runtime` has `Ubuntu 14.04.5 LTS`. Maybe it not big problem, but when I compile Kitura code with MySQL and try launch in runtime container, I have error like this:

`
error while loading shared libraries: libmysqlclient.so.20: cannot open shared object file: No such file or directory
`

I tried find this library, previously I made `apt install libmysqlclient-dev`, and I found `libmysqlclient.so.18`.

So I think that we should have same Ubuntu version in both containers.
